code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2812,3 +2812,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Tich Toch"
 	glass_desc = "Oh god."
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/helianthus
+	name = "Helianthus"
+	description = "A radiant golden mixture of absinthe and hallucinogens. The choice of all true artists."
+	boozepwr = 75
+	color = "#fba914"
+	quality = DRINK_VERYGOOD
+	taste_description = "golden memories"
+	glass_icon_state = "helianthus"
+	glass_name = "Helianthus"
+	glass_desc = "Another reason to cut off an ear..."
+	var/hal_amt = 4
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/helianthus/on_mob_life(mob/living/carbon/drinker, delta_time, times_fired)
+	if(DT_PROB(5, delta_time))
+		drinker.hallucination += hal_amt
```

Comment:
you need to do a log algorithm or a cap to ensure you can't stack hal_amt to ludicrous levels by drinking or forcefeeding others